src/compiler/program.ts

```diff
@@ -226,6 +226,11 @@ namespace ts {
         for (const diagnostic of diagnostics) {
             output += formatDiagnostic(diagnostic, host);
         }
+
+        output += host.getNewLine();

```

Comment:
I don't think this is correct change. We already report summary in tsc watch mode and that's in emitFilesAndReportErrors

Reply:
Right, that makes sense. We should copy that functionality from watch mode then? I feel strongly that the TypeScript compiler should output this information in both modes, by default.